tests/utils.go

```diff
@@ -5317,10 +5285,10 @@ func AddVolumeAndVerify(virtClient kubecli.KubevirtClient, storageClass string,
 	return addVolumeName
 }
 
-func CreateCephPVC(virtClient kubecli.KubevirtClient, name string, size resource.Quantity) *k8sv1.PersistentVolumeClaim {
-	sc, exists := GetCephStorageClass()
+func CreateBlockPVC(virtClient kubecli.KubevirtClient, name string, size resource.Quantity) *k8sv1.PersistentVolumeClaim {
+	sc, exists := GetRWOBlockStorageClass()
```

Comment:
Maybe use the same pattern I have seen in other places, if RWO is not available, try RWX? It could possible RWXBlock is available while RWOBlock is not.

Reply:
first of all if you mean the places I think then its not asking for RWXBlock if RWOBlock is no available it just do that if RWX accessmode was asked (to prevent two ifs).
Also what will it help if we try to create the dv/pvc with a storage class that doesnt support RWO while given explicitly accessmode RWO?

Reply:
Actually I think I flipped the pattern I saw, what I was seeing is
```
res := GetRWOStorageClass(sc)
if GetRWXStorage(sc) {
  res = GetRWXStorage(sc)
}
return res
```
Basically get the RWO storage class, but use the RWX if it is available. Which is not the same here.